src/main/java/dev/emi/trinkets/api/event/TrinketEquipCallback.java

```diff
@@ -0,0 +1,26 @@
+package dev.emi.trinkets.api.event;
+
+import dev.emi.trinkets.api.SlotReference;
+import dev.emi.trinkets.api.Trinket;
+import net.fabricmc.fabric.api.event.Event;
+import net.fabricmc.fabric.api.event.EventFactory;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.item.ItemStack;
+
+public interface TrinketEquipCallback {
```

Comment:
This class is indented using spaces, please use tabs

Reply:
ok